src/app/components/pool/graphs/pool-graphs.html

```diff
@@ -1,9 +1,11 @@
 <bl-banner *ngIf="startTaskFailedError" fixMessage="Edit the start task" [fix]="openEditStartTask">
-    <div message>It seems like the start task is failing on many nodes.
-        ({{startTaskFailedError.failedNodes}}/{{startTaskFailedError.nodeCount}})</div>
+    <div message>It seems like the start task is failing on {{startTaskFailedError.failedStartTaskNodes}} out of {{startTaskFailedError.nodeCount}} nodes.</div>
     <div [other-fix]="rebootFailedNodes" fixMessage="Reboot failed nodes"></div>
     <div *ngIf="isPaasPool" [other-fix]="reimageFailedNodes" fixMessage="Reimage failed nodes"></div>
 </bl-banner>
+<bl-banner *ngIf="unusableNodeError">
```

Comment:
How does this look when both banners are present? Would it be better to combine them and add some conditionals describing whether to have 1 or both messages?

Reply:
When both banners are present it will look like this. I like the separate banners because the start task error has a fix button while the unusable errors don't and it might be confusing if they were combined in the same banner. What are your thoughts? If I keep the two banners I would try to minimize the white space between them.

![image](https://user-images.githubusercontent.com/16728220/72558155-ffe8a100-3856-11ea-8917-999e14fec366.png)

Reply:
Should be fine if we can reduce the padding. Also you probably don't need instructions on what to do for start task as there is already a fix button to communicate what to do